src/codeu/chat/util/UptimeServerInfo.java

```diff
@@ -0,0 +1,28 @@
+package codeu.chat.util;
+
+// why is this class in the util package? Can we combine the two ServerInfo classes in the
```

Comment:
I left this comment before I knew we had to do pull requests but it still stands. I believe the instructions were to put it in the codeu.chat.common package as it is used by both the client and the server.